README.md

```diff
@@ -1,34 +1,32 @@
 # Event Driven Ansible Server
 
+## Installation
+For instructions on how to quickly spin up a preview instance of *eda-server*, please see [INSTALL.md](./INSTALL.md).
+
 ## Setting up a development environment
 
+The below instructions detail how to setup a development environment for _eda-server_. These instructions should be performed on your local system and not on a remote system.
+
 ### Requirements
 
 - [Docker](https://docs.docker.com/engine/install/) or [podman](https://podman.io/getting-started/installation)
 - Docker-compose: `pip install docker-compose`
 - [Taskfile](https://taskfile.dev/installation/)
 - Git
-- Python >= 3.8
-- Node >= v16 (**NOTE:** node v17 does not seem to work with websockets)
+- Python >= 3.9
+- python3-pip
+- python3-devel
+- gcc
+- npm >= v16 (**NOTE:** node v17 does not seem to work with websockets)
 
-NOTE podman users (only for mac and linux):
+**NOTE podman users (only for MacOS and Linux):**
```

Comment:
This section should include as well the tips for the DOCKER_HOST and the service to run the socket.